README.md

```diff
@@ -25,10 +25,17 @@
   - [Nim](#nim)
   - [Pony](#pony)
   - [JavaScript](#javascript)
+  - [AngularJS](#angularjs)
```

Comment:
i would like this guide to only focus on programming languages so I don't want to create a new category for a framework so please just add this under JS section